frontend/components/DashboardAuctionsView.vue

```diff
@@ -18,7 +18,7 @@
             </TextBlock>
         </div>
         <div class="Block max-w-screen-lg">
-            <CollateralTable :collaterals="collaterals" class="overflow-x-scroll" />
+            <CollateralTable :collaterals="collaterals" class="overflow-x-auto" />
```

Comment:
I also fixed a small issue here with the scrollbar. Before the scrollbar was always shown, even if it was not needed. With this new class it only shows once it is actually needed.